### DISCUSSION TITLE
Duino Coin ESP_Code compilation error in Arduino IDE 2.3.2

### DISCUSSION THREAD

**Discussion by Bimbo45y** [revoxhere/duino-coin#1823] [General]
On line 350, got "Compilation error: macro names must be identifiers". Try updating ESP board but error still persist

**Comment by Bimbo45y** [comment]
The problem was solved now. I commented the line 350, 351, and 362. Can compiled and upload to ESP32. But the code become unusable for ESP8266 because the problematic line was ESP8266 related. Also,  what is the yellow triangle warning sign next to the hashrate in the web wallet? Does my ESP32 not running in its full potential? Every one of them has 170kH/s speed. Other problem was in the web wallet sometimes it shows "no miners detected" although my miners are online 24/7. Sometimes only 1 miner detected. Sometimes it is only a few minutes before they show up normally but sometimes it can be as long as half an hour before they (or the one that missing) show up normally. Im worried if my device was running consuming energy but nothing has been send to server.  Why it is happened?

**Comment by jpx13** [comment]
Better using the Arduino ide 1.8.19 (legacy one) as recommanded in the getting started guide as the 2.3.x is known to be buggy...
Don't worry about the yellow triangle, if your hashrate is around 170 kH/s (dual core) then it is fine (the wallet app needs an update)
Even if your miners are not showing in the wallet (which happens rarely  i.e if the server is overloaded ) your miners are still mining and your balance still increasing.
